front/src/modules/people/table/components/GenericEditableCell.tsx

```diff
@@ -0,0 +1,23 @@
+import { EntityFieldMetadata } from '@/ui/table/types/EntityFieldMetadata';
+
+import { GenericEditableTextCell } from './GenericEditableTextCell';
+
+type OwnProps = {
+  entityFieldMetadata: EntityFieldMetadata;
+};
+
+export function GenericEditableCell({ entityFieldMetadata }: OwnProps) {
```

Comment:
all these components should move to ui/table as they are generic. We don't have to call them Generic either.

Reply:
but we can wait for the refactoring to be complete!